README.md

```diff
@@ -1,2 +1,30 @@
-# deploy-prometheus
-deployment pipeline and BOSH operators for prometheus
+# Prometheus Monitoring - BOSH deployment details
+
+Prometheus is an Open source monitoring tool.  A BOSH release was created for it [prometheus-boshrelease](https://github.com/cloudfoundry-community/prometheus-boshrelease).
+
+The BOSH release deploys the following monitoring tools - [Prometheus](<https://prometheus.io/>), [AlertManager](https://github.com/prometheus/alertmanager) and  [Grafana](<https://grafana.com/>).
+
+
+## Usage
+
+### Setting up local BOSH environment
+The instructions for deploying a local BOSH2 director are [here](../setup-local-bosh.md)
```

Comment:
Where should this link go to?

Reply:
oops.   copy-pasta from another repo and should have removed.  well spotted.